modules/sql-engine/src/integrationTest/sql/types/integer/cast_to_integer.test

```diff
@@ -0,0 +1,131 @@
+# name: test/sql/types/decimal/cast_to_integer.test
+# description: Test casting from integer to integer
```

Comment:
description is incorrect

Reply:
why ? integer means not a type but integer range of numbers